server/odc-service/src/main/java/com/oceanbase/odc/service/session/util/DBSchemaExtractor.java

```diff
@@ -196,6 +197,16 @@ private static class OBMySQLRelationFactorVisitor extends OBParserBaseVisitor<Re
 
         private final Set<DBSchemaIdentity> identities = new HashSet<>();
 
+        @Override
+        public RelationFactor visitCreate_index_stmt(Create_index_stmtContext ctx) {
+            for (int i = 0; i < ctx.getChildCount(); i++) {
+                if (i != 2) {
```

Comment:
magic number, which is not acceptable~
btw, how about alter table add index statement